modules/net-lb-app-ext/variables.tf

```
@@ -114,6 +114,7 @@ variable "neg_configs" {
   description = "Optional network endpoint groups to create. Can be referenced in backends via key or outputs."
   type = map(object({
     description = optional(string)
+    project_id  = optional(string, null)
```

The second argument is redundand, `null` is the default.
```suggestion
    project_id  = optional(string)
```